h/services/search_index/_queue.py

```diff
@@ -180,8 +180,8 @@ def sync(self, limit):
 
         LOG.info(dict(counts))
 
-    def count(self, tags=None):
-        query = self._job_query
+    def count(self, tags=None, expired=False):
+        query = self._job_query(expired=expired)
 
         if tags:
             query = query.filter(Job.tag.in_(tags))
```

Comment:
I think this tags part should go down into `_job_query()`. Seems to be the way that function is going.